### DISCUSSION TITLE
I just subscribed to "The GitHub Insider" newsletter, and after confirming, I received two identical emails thanking me for subscribing. How do I report this problem?

### DISCUSSION THREAD

**Discussion by manuelcarriernunes** [community/community#88629] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I just subscribed to "The GitHub Insider" newsletter, and after confirming, I received two identical emails thanking me for subscribing. I expected to receive only one such email. This might be a bug. How do I report this problem?

**Comment by manuelcarriernunes** [comment]
Thanks for trying to answer my question, but the answer doesn't help me solve my problem. The following are the reasons why:

- The page at https://support.github.com was where I started before posting this question. The following explains the path I took to go from https://support.github.com to https://github.com/orgs/community/discussions/categories/general:
    1. I started on https://support.github.com.
    2. On https://support.github.com, the only category that seemed to fit my problem was "Product Feedback." Clicking on "Product Feedback" brought me to https://github.com/orgs/community/discussions.
    3. On https://github.com/orgs/community/discussions, the only category that seemed to fit my problem was "General." This is when I wrote this question.
- Going to https://support.github.com/tickets/personal gave me the page shown in the screenshot below.

![image](https://github.com/community/community/assets/3910632/206b5765-35c6-4365-8ec3-30b16e522e2b)

Your suggestion to create a ticket did make me curious about how to create a ticket, so I went to https://support.github.com and typed, "How do I create a support ticket?" The results included a link to the following page: https://docs.github.com/en/support/contacting-github-support/creating-a-support-ticket. On that page, it says, "If your account uses GitHub Free, you can speak to GitHub users and staff on the [GitHub Community discussions](https://github.com/orgs/community/discussions) for most issues, and you can contact GitHub Support to report account, security, and abuse issues."

Sadly, I don't think my problem is an account, security, or abuse issue, so writing this question was the only thing I could do as someone whose account uses GitHub Free.

**Reply by mishmanners** [reply]
Going to https://support.github.com/tickets/personal appears to give an error if you don't have any tickets logged 🤷 

Product Feedback is when you want to give feedback on a particular product, for example GitHub Actions, in which case these Discussions are a good place to go.

When it comes to the original issue, did you receive two identical emails to the exact same email address? If so, let me know and then I can mark this discussion as a bug so the appropriate team will know there's a bug and can look into it, rather than product feedback which is usually for discussing new features within products.

**Reply by manuelcarriernunes** [reply]
Thanks for taking the time to help.

Yes; I received two identical emails at the same address.
